ledger-api/grpc-definitions/com/daml/ledger/api/v1/transaction_filter.proto

```diff
@@ -72,8 +84,34 @@ message InterfaceFilter {
 
   // Whether to include a ``create_arguments_blob`` in the returned
   // ``CreateEvent``.
-  // Use this to access the complete contract data in your API client
+  // Use this to access the complete contract create argument data in your API client
+  // for submitting it as a disclosed contract with future commands.
+  // Deprecated in favor of ``include_contract_create_payload``. If the latter field
+  // is set, ``include_create_arguments_blob`` is ignored.
+  // Optional
+  bool include_create_arguments_blob = 3 [deprecated = true];
+
+  // Whether to include a ``contract_create_payload`` in the returned
+  // ``CreateEvent``.
+  // Use this to access the complete contract create payload in your API client
+  // for submitting it as a disclosed contract with future commands.
+  // Optional
+  bool include_contract_create_payload = 4;
+}
+
+// This filter matches contracts of a specific template.
+message TemplateFilter {
+
+  // A template for which the payload should be included in the response.
+  // The ``template_id`` needs to be valid: corresponding template should be defined in
+  // one of the available packages at the time of the query.
+  // Required
+  Identifier template_id = 1;
+
+  // Whether to include a ``contract_create_payload`` in the returned
+  // ``CreateEvent``.
+  // Use this to access the complete contract payload in your API client
```

Comment:
```suggestion
  // Use this to access the contract create payload in your API client
```
The usage of "complete" in the above descriptions seems tautological. I'd remove it in favor for consistently using the same way to refer to the "contract_create_payload" (or if we go for it the "create_event_payload" or "create_event_blob")